src/layouts/hass-loading-screen.ts

```diff
@@ -16,28 +16,32 @@ import { HomeAssistant } from "../types";
 
 @customElement("hass-loading-screen")
 class HassLoadingScreen extends LitElement {
-  @property({ type: Boolean }) public rootnav? = false;
+  @property({ type: Boolean }) public toolbar = true;
```

Comment:
If we do this, we can remove `layouts/loading-screen.ts`, which is the loading screen without a toolbar.

Reply:
Let me turn it around then, so we can do `<hass-loading-screen no-toolbar></hass-loading-screen>`

Reply:
sounds good